R/constructor.R

```diff
@@ -1,9 +1,11 @@
-new_bagger <- function(model_df, imp, control, cost, base_model, blueprint) {
+new_bagger <- function(model_df, imp, control, cost, base_model, blueprint,
+                       call = rlang::caller_env()) {
 
   if (!is_tibble(model_df)) {
-    cli::cli_abort("`model_df` should be a tibble.")
+    cli::cli_abort("{.arg model_df} should be {.cls tibble}.", call = call)
   }
 
+  # TODO extend to use mode from model object(s)
```

Comment:
We could eventually extend baguette to censored regression models. There is a lot of code that looks at the data to get the mode or assumes that they are either regression or classification.